roles/ceph-osd/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: install dependencies
+  apt: pkg=parted
+       state=present
```

Comment:
This is okay for now, but if there is ever a second package to install this should get moved out into the defaults as a list.